README.md

```diff
@@ -211,13 +211,31 @@ Rules for contributions can be found in [CONTRIBUTING.md](CONTRIBUTING.md)
 Authors
 -------
 
-### Maintainers and Core Developers
-
-- Benjamin Worpitz (original author)
-- Rene Widera
+### Maintainers* and Core Developers
+
+- Benjamin Worpitz* (original author)
+- Dr. Sergei Bastrakov*
+- Mat Colgrove
```

Comment:
Note: [this commit](https://github.com/alpaka-group/alpaka/commit/97823c78edd1348406b193de2616460748f5b1a0)

Reply:
I would move Mat to Contributors because he is not actively developing on alpaka.

Reply:
Okay, moved.